bin/seeg-ct.sh

```diff
@@ -22,7 +22,7 @@
 # parameters of image processing can be tuned while looking at the results.
 
 
-if [ -z "$SUBJECT" || -z "$CT" ]
+if [ -z "$SUBJECT" ] || [ -z "$CT" ]
```

Comment:
maybe the original syntax requires double square brackets? this is fine anyway, if it's working